erpnext/stock/stock_ledger.py

```diff
@@ -701,14 +701,18 @@ def get_moving_average_values(self, sle):
 						sle.voucher_type, sle.voucher_no, self.allow_zero_rate,
 						currency=erpnext.get_company_currency(sle.company), company=sle.company)
 
-	def update_fifo_values(self, sle):
+	def update_fifo_lifo_values(self, sle):
```

Comment:
```suggestion
	def update_queue_values(self, sle):
```

Felt the named method didn't make much sense. Also, how about adding type hints? I'm not sure what `sle` is supposed to be without reading the code above or below.

Reply:
+1 on typing, right now sles are either dictionary or _dict or actual sle document, often all interchangeably 🥲